webp_backport/src/main/java/de/marcreichelt/webp_backport/WebPBackport.java

```diff
@@ -38,17 +38,26 @@ static boolean isIsWebpSupportedNatively() {
     }
 
     /**
+     * Check if a given byte array is a WebP image
+     *
+     * @param encoded the byte array to check
+     * @return true if the byte array is recognized as a WebP image
+     */
+    public static boolean isWebP(byte[] encoded) {
+        return FileSignatureChecker.checkForWebP(encoded);
+    }
+
+    /**
      * Decodes a WebP image.
      *
      * @param encoded The encoded WebP data (e.g. from a stream, resource, etc.).
      * @return The decoded image, or {@code null} if it could not be decoded.
```

Comment:
The code of this method has been updated, so now it does a completely different thing (also supports other image formats now). The documentation should match.